TestsCalculator/CalculatorPageTests.cs

```diff
@@ -195,5 +195,30 @@ public void CheckDefaultStartDate()
             //Assert
             Assert.AreEqual(DateTime.Today.ToString("d/M/yyyy"), calculatorPage.StartDate);
         }
+
+        [Test]
+        public void CheckSettingsBtnDisplayed()
```

Comment:
No need to add tests to verify control exists, if you will click it next. The 'click' action will fail, if the control is not shown.